src/protocols/rdp/client.h

```diff
@@ -87,6 +87,11 @@
  */
 #define GUAC_RDP_AUDIO_BPS 16
 
+/**
+ * The default interval to send the audio input packet to a remote server, in ms.
+ */
+#define GUAC_RDP_AUDIO_INPUT_SEND_DEFAULT_INTERVAL 5
```

Comment:
I don't think this should be needed if the outbound audio data rate is correctly throttled based on the server-requested frames per packet.